benchmarking/src/bin/benchmark.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022-2023 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+//! Run all of the examples as benchmarks and report the results in a table.
+
+use std::time::Duration;
+
+use benchmarking::{
+    run::{get_examples, BenchmarkResult},
+    time_bench::{compile_flyvy_bin, compile_time_bin, REPO_ROOT_PATH},
+};
+use clap::Parser;
+
+#[derive(clap::Subcommand, Clone, Debug, PartialEq, Eq)]
+enum Command {
+    Verify {
+        /// Time limit for verifying each file.
+        #[arg(long, default_value = "60s")]
+        time_limit: humantime::Duration,
+    },
+}
+
+#[derive(clap::Parser, Debug)]
+struct App {
+    /// Command to run
+    #[command(subcommand)]
+    command: Command,
+}
+
+fn benchmark_verify(time_limit: Duration) -> Vec<BenchmarkResult> {
```

Comment:
Is this used anywhere else? Why is it separate from `App::exec`?

Reply:
This is separate to make it easier to see the control flow when we have similar functions for benchmarking qalpha and bounded model checking. (And also to make it obvious what needs to be implemented to add those.)